src/test/java/io/strimzi/kafka/quotas/StaticQuotaCallbackTest.java

```diff
@@ -161,6 +173,229 @@ void staticQuotaMetrics() {
         assertEquals(expectedMbeanName, name.getMBeanName(), "unexpected mbean name");
     }
 
+    @Test
+    void shouldSetQuotaFactorToZeroIfHardLimitBreached() {
+        //Given
+        final Map<String, Long> diskUsage = Map.of("Disk one", 20L);
+        target.configure(Map.of(
```

Comment:
This should maybe done in the `setup` method, I've left it local to the new tests for now.